src/config/ConfigManager.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+export default class ConfigManager {
+
+    constructor(config = {}) {
```

Comment:
This `config` parameter is never used. Should it be the fallback below in case global.config is null?

Reply:
Fixed this so the config object is based off of a combination of what is passed in and any global.CONFIG that may be available.